gradle.properties

```diff
@@ -29,7 +29,7 @@ gradleMkdocsPluginVersion=1.1.0
 jacocoVersion=0.8.2
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.6.1
+connectorVersion=0.6.2-SNAPSHOT
```

Comment:
This change seems unrelated to the purpose of the PR. Please, try to avoid conflating changes like that as it is confusing.